components/tokens/gulpfile.js

```diff
@@ -15,7 +15,7 @@ const concat = require('gulp-concat');
 const del = require('del');
 const StyleDictionary = require('style-dictionary').extend('config.js');
 
-function clean() {
+async function clean() {
```

Comment:
@Rajdeepc I think you're looking at docs for `del` 7.x.  We're locked in at 5.x for no need to make any changes to these functions.